tests/src/test/java/edu/kit/kastel/mcse/ardoco/core/tests/TestUtil.java

```diff
@@ -77,6 +79,23 @@ public static int calculateTrueNegativesForTLR(ArDoCoResult arDoCoResult, Explic
         return confusionMatrixSum - (truePositives + falsePositives + falseNegatives);
     }
 
+    /**
+     * Calculates the number of true negatives based on the given {@link ArDoCoResult} and the calculated {@link EvaluationResults evaluation
+     * results}.
+     * Uses the total sum of all sentences in the {@link ArDoCoResult} and then substracts the true positives, false positives, and false negatives.
+     *
+     * @param arDoCoResult   the output of ArDoCo
+     * @param truePositives  nr of true positives
+     * @param falsePositives nr of false positives
+     * @param falseNegatives nr of false negatives
+     * @return the number of true negatives
+     */
+    public static int calculateTrueNegativesForInconsistencies(ArDoCoResult arDoCoResult, int truePositives, int falsePositives, int falseNegatives) {
+        int numberOfSentences = arDoCoResult.getText().getSentences().size();
+        return numberOfSentences - (truePositives + falsePositives + falseNegatives);
+
+    }
```

Comment:
see above. This can be encapsulated to a class, to ensure extensions in the future